docs/topics/email.txt

```diff
@@ -529,6 +538,16 @@ To specify this backend, put the following in your settings::
 This backend is not intended for use in production -- it is provided as a
 convenience that can be used during development.
 
+..method:: format_message(message)
```

Comment:
Wondering if we can just adjust the doc above to say it subclasses console backend. (?)